### PR TITLE
Update diff_drive controller parameters

### PR DESCRIPTION
**Pull request by Wiktor-99**
PR addresses the #459

### CODE REVIEW

**Review by bmagyar — APPROVED** [review]

**Review by saikishor — APPROVED** [review]

File: gz_ros2_control_demos/config/diff_drive_controller.yaml [hunk 1]
```diff
@@ -36,19 +36,17 @@
     # Whenever a min_* is unspecified, default to -max_*
     linear.x.has_velocity_limits: true
```
Comment (christophfroehlich) [inline gz_ros2_control_demos/config/diff_drive_controller.yaml:37]: velocity and acceleration are deprecated similarly (just no output if the value is true, i.e., the default)

**Review by ahcorde — APPROVED** [review]

**Comment by ahcorde** [comment]
Thank your @Wiktor-99

**Review by christophfroehlich — CHANGES_REQUESTED** [review]
thanks @Wiktor, but this isn't your first issue in ros-controls, right? 😅

**Comment by christophfroehlich** [comment]
@mergifyio backport jazzy

**Comment by Wiktor-99** [comment]
> thanks @wiktor, but this isn't your first issue in ros-controls, right? 😅

You're right. I didn't know that the good first issue policy is so strict.😂
Since I've started, I believe I can fix the rest of the parameters.

**Comment by christophfroehlich** [comment]
it's not strict. But sometimes it's hard to find easy tasks to help onboarding of new contributers. If experienced developers also take these tasks although not urgent, then nothing is left for them 🫣

**Comment by Wiktor-99** [comment]
Gotcha, next time I'll try to pick more complicated issues

**Pull request merged**